### ISSUE TITLE
Harden and strip the distribution binary's Qt

### ISSUE THREAD

**Issue by special** [ricochet-im/ricochet#392]
We should harden and strip down the Qt used for distributed binaries, to get rid of things Ricochet doesn't need and things that might be dangerous to have around.

One example that came up is getting rid of V4_ENABLE_JIT and the PCRE equivalent, so there isn't even JIT capable code built into the binary.

**Comment by ioerror** [comment]
Agreed - we can easily do this for the bundled Ricochet but not so easily for the other builds (eg: debian).

**Comment by special** [comment]
Qt 5.8 has the 'Lite' project that allows much deeper configuration of what features are included in a Qt build. Unclear when we'll be able to switch the binaries over to 5.8, however.
